server/src/controllers/auth.controller.js

```diff
@@ -39,9 +39,36 @@ const signup = ( req, res) => {
         });
     });
 };
+const signin = (req,res) => {
+    User.findOne({ email: req.body.email})
+    .exec ( (error, user) => {
+        if(error)   return res.status(400).json(responseHandler( false, 400, "Something went wrong...!", null ));
```

Comment:
I think we do not need this error confirmation line, Because in the below else condition handles this error as well.

Reply:
ok